### DISCUSSION TITLE
Migration to Fleet API has started?

### DISCUSSION THREAD

**Discussion by thierryvt** [alandtse/tesla#839] [General]
Literally added this integration yesterday, which worked for all of 30 minutes or so. With debug mode I found this in the logs:

> 2024-01-27 10:33:43.035 DEBUG (MainThread) [teslajsonpy.connection] get: https://owner-api.teslamotors.com/api/1/vehicles {}
> 
> 2024-01-27 10:33:43.209 DEBUG (MainThread) [teslajsonpy.connection] 412: {"response":null,"error":"Endpoint is only available on fleetapi. 
> Visit https://developer.tesla.com/docs for more info","error_description":""}

Anyone else running into this or do I just have shit luck?

Edit: just found this discussion here: https://github.com/alandtse/tesla/issues/774 
Seems like Tesla is indeed turning off the old API and this integration is now deprecated.

**Comment by skipishere** [comment]
As you mentioned, this is spoken about at length in https://github.com/alandtse/tesla/issues/774
